### PR TITLE
JBIDE-19266 Batch validation problems QuickFixes

### PR DESCRIPTION
**Pull request by dazarov**

### CODE REVIEW

File: common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java [hunk 1]
```diff
@@ -101,5 +101,21 @@ public static String getDefaultBeanName(String className){
...
 	}
-
+	
+	/**
+	 * Converts name of Bean to Java Class Name
+	 * 
+	 * @param beanName is short name or fully qualified name
+	 * @return Java Class Name
+	 */
+	public static String getClassName(String beanName){
```
Comment (alexeykazakov) [inline common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java]: We should add a unit test for this method.

File: common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java [hunk 2]
```diff
@@ -101,5 +101,21 @@ public static String getDefaultBeanName(String className){
...
+		int lastDotPosition = beanName.lastIndexOf("."); //$NON-NLS-1$
+		if(lastDotPosition < 0 || beanName.length() < lastDotPosition){
```
Comment (alexeykazakov) [inline common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java]: beanName.length() is never < lastDotPosition

File: common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java (hunk 1 above)
Comment (dazarov) [inline common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java]: Done

File: common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java (hunk 2 above)
Comment (dazarov) [inline common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java]: Done

**Comment by alexeykazakov** [comment]
I have already pushed the previous version of this PR: https://github.com/jbosstools/jbosstools-base/commit/a22167c8235be4df0fde903a2d797d54690cd8d7
I'm closing it now but we should create a new PR for JBIDE-19266

**Pull request closed without merging**